sys/auto_init/screen/auto_init_lvgl.c

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (C) 2020 Inria
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     sys_auto_init
+ * @{
+ * @file
+ * @brief       initializes lvgl high level GUI api
+ *
+ * @author      Alexandre Abadie <alexandre.abadie@inria.fr>
+ * @}
+ */
+
+#include "log.h"
+
+#include "lvgl_riot.h"
+
+#include "screen_dev.h"
+#include "disp_dev.h"
+
+#if MODULE_TOUCH_DEV
+#include "touch_dev.h"
+#endif
+
+static screen_dev_t s_screen;
+extern disp_dev_reg_t *disp_dev_reg;
+
+#if MODULE_TOUCH_DEV
+static void _touch_event_callback(void *arg)
+{
+    (void)arg;
+    lvgl_wakeup();
+}
+#endif
+
+void auto_init_lvgl(void)
+{
+    LOG_DEBUG("[auto_init_screen] initializing lvgl\n");
+
+    /* Only a single screen is supported by lvgl */
+    disp_dev_reg_t *disp_dev = disp_dev_reg_find_screen(0);
```

Comment:
Does it make sense to replace the hardcoded `0` by a define? One here and one for the touchscreen, both default 0, but a user can override it to a different display ID.

Reply:
Agreed and added `LVGL_SCREEN_DEFAULT` constant for this.